tests/cases.py

```diff
@@ -2067,6 +2070,32 @@ def test_increasing(self):
         else:
             self.assertLessEqual(y, x)
 
+    def test_expectation(self):
+        """Test the numeric expectation is close to the closed form one."""
+        if not self.check_expectation:
+            self.skipTest("no implementation of closed-form expectation")
+        simulated = self.instance.numeric_expected_value(
+            num_candidates=self.num_candidates,
+            num_samples=10000,
+        )
```

Comment:
how expensive is this? should this be run as part of CI? Can we fix the random seed to make CI reproducible?

Reply:
It doesn't appear to be in the top 20 slowest tests, so I think it's fine